packages/docusaurus-plugin-content-blog/src/__tests__/feed.test.ts

```diff
@@ -196,3 +228,95 @@ describe.each(['atom', 'rss', 'json'])('%s', (feedType) => {
     fsMock.mockClear();
   });
 });
+
+describe('Test defaultCreateFeedItems', () => {
```

Comment:
That test looks super complex to me and I don't understand why.

Just call `createBlogFeedFiles` and take a snapshot: we'll review the snapshot and validate it contains what we expect

Reply:
I think that I don't need to add new test case; instead I just need to update the snapshot. Am I correct ?